### PR TITLE
Task/add perseo log level env

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix for https://github.com/telefonicaid/perseo-core/issues/178

  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     011-May-2022 13:00:40.537 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/perseo-core.war] has finished in [15,015] ms
11-May-2022 13:00:40.563 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     011-May-2022 13:00:40.740 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in [14678] milliseconds
time=2022-05-11T13:00:41.267Z | lvl=INFO | from= | corr= | trans= | srv= | subsrv= | op=doPut | comp=perseo-core | msg=changing log level to INFO
  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0
time=2022-05-11T13:00:43.815Z | lvl=INFO | from=172.17.0.14 | corr=n/a | trans=248f1188-89b6-4f68-8c20-7ccb48c77477 | srv=n/a | subsrv=n/a | op=updateAll | comp=perseo-core | msg=put rules+contexts: 10

### CODE REVIEW

**Comment by AlvaroVega** [comment]
This is a low cost implementation based on API to change log level

**Review by fgalan — APPROVED** [review]
LGTM

File: perseo_core-entrypoint.sh [hunk 1]
```diff
@@ -30,6 +30,10 @@ touch /var/log/perseo/perseo-core.log
 
 ln -snf /dev/stdout /var/log/perseo/perseo-core.log &
 
+if [ "$PERSEO_LOG_LEVEL" ]; then
+    (sleep 10; curl -X PUT 'http://localhost:8080/perseo-core/admin/log?level='$PERSEO_LOG_LEVEL) &
```
Comment (cesarjorgemartinez) [inline perseo_core-entrypoint.sh:34]: This type of technique may involve undesirable end-of-run conditions, could there be a better technique?

Comment (AlvaroVega) [reply perseo_core-entrypoint.sh:34]: Maybe you can provide one.

Comment (fgalan) [reply perseo_core-entrypoint.sh:34]: @cesarjorgemartinez this PR provides a solution that suffices the implementetion of the env var and it has been  merged.

Of course, if you have a better solution, please do the corresponding PR so we can merge :)

**Pull request merged**